### ISSUE TITLE
Scio compatibility with Scala 2.13

### ISSUE THREAD

**Issue by jto** [spotify/scio#1553]
Scala 2.13 should be released in the next few weeks and could potentially break source compatibility (because of the changes in the Collection API). 

Assuming Scio works properly with Scala 2.12 (https://github.com/spotify/scio/issues/1275), we should make sure it is also compatible with 2.13 and cross-publish binaries for Scala 2.11, 2.12 and 2.13.

- [x] [cats](https://mvnrepository.com/artifact/org.typelevel/cats-core_2.13/2.0.0)
- [x] algebird
- [x] chill
- [x] [jackson-module-scala](https://mvnrepository.com/artifact/com.fasterxml.jackson.module/jackson-module-scala_2.13/2.9.9)
- [x] [protobuf-generic](https://mvnrepository.com/artifact/me.lyh/protobuf-generic_2.13/0.2.5)
- [x] [case-app](https://mvnrepository.com/artifact/com.github.alexarchambault/case-app_2.13/2.0.0-M9)
- [x] [magnolia](https://mvnrepository.com/artifact/com.propensive/magnolia_2.13/0.11.0) #2192 #2241
- [x] [scalatest](https://mvnrepository.com/artifact/org.scalatest/scalatest_2.13/3.0.8)
- [x] [scalacheck](https://mvnrepository.com/artifact/org.scalacheck/scalacheck_2.13/1.14.0)
- [x] [scalacheck-shapeless](https://mvnrepository.com/artifact/com.github.alexarchambault/scalacheck-shapeless_1.14_2.13/1.2.3)
- [x] [shapeless-datatype](https://mvnrepository.com/artifact/me.lyh/shapeless-datatype-core_2.13/0.2.0)
- [x] [shapeless](https://mvnrepository.com/artifact/com.chuusai/shapeless_2.13/2.3.3)
- [x] annoy4s https://github.com/annoy4s/annoy4s/pull/15
- [x] [breeze](https://mvnrepository.com/artifact/org.scalanlp/breeze_2.13/1.0)
- [x] [circe](https://mvnrepository.com/artifact/io.circe/circe-core_2.13/0.12.1)
- [x] [parquet-avro-extra](https://mvnrepository.com/artifact/me.lyh/parquet-avro-extra_2.13/0.2.3)
- [x] featran
- [x] kantan.csv https://github.com/nrinaudo/kantan.csv/pull/187
- [ ] spark

**Comment by regadas** [comment]
we might be a bit blocked by some of our dependencies, i.e: `algebird`, `breeze`, etc. Their update cycle is a bit slow.

**Comment by nevillelyh** [comment]
Added a checklist. We can expand as we go and fix some of them when we get a chance.

**Comment by jto** [comment]
Should we have a separate module for Spark ?
Spark is probably going to lag behind, maybe we could have everything published for 2.13 but a Spark module ?

**Comment by nevillelyh** [comment]
All deps except spark are on 2.13 now. So should probably split it into a separate module.

**Comment by regadas** [comment]
Started working on [this](https://github.com/regadas/scio/tree/scala-2.13). This will imply dropping 2.11 #2599.

**Comment by regadas** [comment]
Closed via #1553
